.gitignore

```diff
@@ -0,0 +1,17 @@
+# Binaries for programs and plugins
```

Comment:
Please remove these unrelated changes

Reply:
I did so by reverting the gitignore commit, sorry for pushing unrelated changes.